include/maliput/common/logger.h

```diff
@@ -238,10 +231,16 @@ class Logger {
   // Minimum level of messages to be log.
   logger::level level_{logger::level::info};
 
-  // Create a string from a list of strings.
-  // @param v Is a list of strings of fmt type. See https://github.com/fmtlib/fmt.
-  // @return A string created from the list `v` using fmt library.
-  const std::string format(const std::vector<std::string>& v) const;
+  // Create a string from a list of arguments.
+  // @tparam Args Variadic arguments to generate the list of arguments.
+  // @param args Is an argument list representing objects with an already defined serialization applied.
+  // @return A string obtained by concatenating the arguments.
+  template <typename... Args>
+  const std::string format(Args&&... args) const {
+    std::string result;
+    ((result += args), ...);
+    return result;
+  }
```

Comment:
To note, I changed the signature of the method because it wasn't necessary for the implementation this PR adds, however this is breaking API and ABI. I could try to not to break it and keep same signature if needed.

Reply:
The effort to keep the exact same API would be non-trivial?

Reply:
I'm fine with this if it's less effort going forward.

Reply:
No added. See https://github.com/maliput/maliput/pull/568/commits/2ea5f85623ab00ae68d17609fd3c10a398c5b06f